lwgenerate/azure/azure.go

```diff
@@ -97,6 +97,10 @@ func (args *GenerateAzureTfConfigurationArgs) validate() error {
 		return errors.New("audit log or config integration must be enabled")
 	}
 
+	if (args.ActivityLog || args.Config) && args.SubscriptionID == "" {
```

Comment:
Can we also add `EntraIdActivityLog` to this check, please? All azure integrations use the `azurerm` provider

Reply:
+1

Reply:
Updated.